### DISCUSSION TITLE
Remove a vehicle I no longer own?

### DISCUSSION THREAD

**Discussion by loverso** [alandtse/tesla#865] [Q&A]
I just replaced an older Tesla with a new one. The Tesla app no longer shows the old car. The integration picked up the new car just fine, but the old car is still a device in HA. How do I remove the old car device and entities?

I also see a loaner Model 3 device, for a car I had as a loaner back in October!

I searched but couldn't find an answer here or in any of the general HA documentation. For the moment I've disabled the old car device. I see suggestions that I should delete the integration and re-install it, which seems drastic.

**Comment by mbichler78** [comment]
I am also interested as i face the same issue

**Reply by FalconUK** [reply]
I have the same question. I don't have my model 3 anymore, but I can't find a way to remove it from the integration.

**Comment by FalconUK** [comment]
I have the same question. I don't have my model 3 anymore, but I can't find a way to remove it from the integration.
